react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/RNPaywallsModule.kt

```diff
@@ -0,0 +1,44 @@
+package com.revenuecat.purchases.react.ui
+
+import androidx.fragment.app.FragmentActivity
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import com.revenuecat.purchases.hybridcommon.ui.presentPaywallFromFragment
+
+class RNPaywallsModule(reactContext: ReactApplicationContext) :
```

Comment:
Hmm I wonder, should this be internal? Not sure if the client can access these TBH

Reply:
Just made it internal and it works fine, so I think we are good yes, it's accessed from the `RNPaywallsPackage` only